src/components/sections/Landing/Body/Mint/index.tsx

```diff
@@ -1,28 +1,161 @@
+import { notify } from '../../../../../utils/notify';
+
+import { useWeb3Context } from '../../../../../context/WalletConnect';
+import WalletModal from '../../../../molecules/Modals/WalletModal/index';
+import MintModal, { IMintModalProps } from '../../../../molecules/Modals/MintModal/index';
+import { useModals } from '../../../../../context/Modal';
+
 import s from './Mint.module.scss';
-import nftExample from '../../../../../assets/img/sections/landing/mint/nft-example.png';
+
+import nft1 from '../../../../../assets/img/sections/landing/mint/nft-1.png';
+import nft2 from '../../../../../assets/img/sections/landing/mint/nft-2.png';
+import nft3 from '../../../../../assets/img/sections/landing/mint/nft-3.png';
+import nft4 from '../../../../../assets/img/sections/landing/mint/nft-4.png';
+import { useCallback, useEffect, useState } from 'react';
+
+const TIME_FOR_UPDATE = 20000;
 
 const Mint: React.FC = () => {
+  const { init, sendEth } = useWeb3Context();
+  const { setModal } = useModals();
+
+  const [mintModalProps, setMintModalProps] = useState({
+    img: '',
+    type: 'COMMON',
+    txHash: '',
+  } as IMintModalProps);
+
+  const getInfoAboutTx = useCallback(
+    async (txHash: string) => {
+      const headers = await fetch(`https://deeznuts.rocknblock.io/api/v1/payments/${txHash}`);
+      const data = await headers.json();
+
+      if (data.status === 'SUCCESS') {
+        setMintModalProps({
+          type: data.rarity === 'common' ? 'COMMON' : 'LEGENDARY',
+          img: data.image,
+          txHash,
+        });
+        setModal('mintmodal');
+      }
+
+      notify(
+        `Tx ${txHash.slice(0, 5)}...${txHash.slice(-5)}: ${data.status
+          .replaceAll('_', ' ')
+          .toLowerCase()}`,
+        'info',
+      );
+    },
+    [setModal],
+  );
+
+  const mintNft = async (wallet: 'MetaMask' | 'WalletConnect') => {
+    const info = await init(wallet);
+
+    if (!info) {
+      notify('No Web3 Provider! Please install or download MetaMask', 'error');
+    }
+
+    if (info.code === 3) {
+      notify(`${info.message.message} Connect your wallet!`, 'error');
+    }
+
+    if (info.code === 4) {
```

Comment:
лучше использовать `includes` если результат будет одинаковый:

```suggestion
    if ([4,404].includes(info.code)) {
      notify(info.message.text, 'error');
      return;
    }
```